src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -85,8 +85,14 @@ def update_data(self):
             if not ent:
                 continue
 
-            # Example output: oid:0x3a000000000608
-            bridge_port_id = ent["SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID"][6:]
+            bridge_port_id = ""
+            try:
+                # Example output: oid:0x3a000000000608
+                bridge_port_id = ent["SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID"][6:]
+            except:
```

Comment:
>except [](http://example.com/codeflow?start=12&length=6)

Catch more specific exception, or use if-else to check.

Reply:
Fixed, only catch KeyError